.travis.yml

```diff
@@ -19,94 +19,49 @@ matrix:
     - jdk: oraclejdk8
       env:
         - DESC="eclipse-cs"
-        - CMD_="cd sevntu-checks && mvn -e clean install -Dmaven.test.skip=true -Dcheckstyle.skip=true -Dcobertura.skip=true && cd .. "
-        - CMD0="git clone git://git.code.sf.net/p/eclipse-cs/git eclipse-cs-git"
-        - CMD1="&& cd eclipse-cs-git/ && git checkout 8.5.1"
-        - CMD2="&& cd net.sf.eclipsecs.parent/ && mvn -e install"
-        - CMD3="&& cd ../../"
-        - CMD4="&& cd eclipsecs-sevntu-plugin && mvn -e verify && mvn -e javadoc:javadoc"
-        - CMD=$CMD_" && "$CMD0$CMD1$CMD2$CMD3$CMD4
-        - PREINSTALL="true"
+        - CMD="./.ci/travis.sh eclipse-cs"
 
     # maven-plugin
     - jdk: oraclejdk8
       env:
         - DESC="maven-plugin"
-        - CMD0="cd sevntu-checks && mvn -e clean install -Dmaven.test.skip=true -Dcheckstyle.skip=true -Dcobertura.skip=true && cd .. "
-        - CMD=$CMD0" && ""cd sevntu-checkstyle-maven-plugin && mvn -e verify && mvn -e javadoc:javadoc"
+        - CMD="./.ci/travis.sh maven-plugin"
 
     # idea-extension
     - jdk: oraclejdk8
       env:
         - DESC="idea-extension"
-        - CMD0="cd sevntu-checks && mvn -e clean install -Dmaven.test.skip=true -Dcheckstyle.skip=true -Dcobertura.skip=true && cd .. "
-        - CMD=$CMD0" && ""cd sevntu-checkstyle-idea-extension && mvn -e verify && mvn -e javadoc:javadoc"
+        - CMD="./.ci/travis.sh idea-extension"
 
     # sonar-plugin
     - jdk: oraclejdk8
       env:
         - DESC="sonar-plugin"
-        - CMD0="cd sevntu-checks && mvn -e clean install -Dmaven.test.skip=true -Dcheckstyle.skip=true -Dcobertura.skip=true && cd .. "
-        - CMD=$CMD0" && ""cd sevntu-checkstyle-sonar-plugin && mvn -e verify && mvn -e javadoc:javadoc"
```

Comment:
@romani is it possible this `" && ""` was causing a bug for the files not to be validated?

Reply:
By shell syntax it looks right - just string concatenation

Reply:
Looks like you about right , https://travis-ci.org/sevntu-checkstyle/sevntu.checkstyle/jobs/324756666#L441 ,  and https://github.com/sevntu-checkstyle/sevntu.checkstyle/blob/master/.travis.yml#L48 , command ci config were cut, partly executed.